test/dpul_collections/indexing_pipeline_test.exs

```diff
@@ -99,6 +99,19 @@ defmodule DpulCollections.IndexingPipelineTest do
       assert IndexingPipeline.get_figgy_resource!(ephemera_folder_id).id == ephemera_folder_id
     end
 
+    test "get_figgy_resource!/1 errors when the resource isn't there" do
```

Comment:
Where'd this test come from?

Reply:
I added this, it seemed like the error case wasn't tested and then I used it as a model to check the behavior of the new function. Do you think it's redundant?

Reply:
Not at all, it just seemed unrelated so I was curious if there was something in developing this that brought a bug to light.